enzyme/index.d.ts

```diff
@@ -541,14 +541,14 @@ export interface MountRendererProps {
  * @param node
  * @param [options]
  */
-export function shallow<P, S>(node: ReactElement<P>, options?: ShallowRendererProps): ShallowWrapper<P, S>;
+export function shallow<P, S>(node: ReactElement<P>, options?: ShallowRendererProps): ShallowWrapper<P, any>;
 
```

Comment:
Sorry I should have been clearer. What I'm saying is you get the same return type (```ShallowWrapper<P,any>```) regardless of what you supply for the ```S``` type argument, so there's no point in having it on the ```shallow``` function anymore.

May I suggest adding an overload like the following to preserve the existing strong-typing, while having the option to omit type parameters.

```ts
declare function mount<P>(node: ReactElement<P>, options?: MountRendererProps): ReactWrapper<P, any>;
declare function mount<P, S>(node: ReactElement<P>, options?: MountRendererProps): ReactWrapper<P, S>;
```

If ```S``` is explicitly supplied, you get a typed return value. If it's not supplied, you get ```any```. This way, existing users of this file who have already supplied type parameters won't get ```any``` types leaking into their code.